modules/govuk_jenkins/templates/jobs/data_sync_complete_integration.yaml.erb

```diff
@@ -24,6 +24,9 @@
            ssh deploy@$(govuk_node_list -c backend --single-node) 'cd /var/apps/publisher ; govuk_setenv publisher bundle exec rake editions:requeue_scheduled_for_publishing'
            # Publish any pre-production finders from Specialist Publisher Rebuild.
            ssh deploy@$(govuk_node_list -c backend --single-node) 'cd /var/apps/specialist-publisher-rebuild ; govuk_setenv specialist-publisher-rebuild bundle exec rake publishing_api:publish_finders rummager:publish_finders'
+           # Clear the Sidekiq queues to stop any residual 'stale' jobs running
+           ssh deploy@redis-1.backend.integration.publishing.service.gov.uk 'redis-cli flushall'
```

Comment:
I wondered about doing that. That'll flush all the redissesses though won't it? I'm not sure what the others are used for. Are we ok with that?